lists2safebrowsing.py

```diff
@@ -500,6 +544,32 @@ def main():
 
     publish_to_cloud(config, chunknum)
 
+    shavar_prod_lists_branch = {
+      'name': '70',
+      'commit': {
+        'sha': '8ffa3c6d702002e9a357dd0042848c9418e0ef8a',
+        'url': 'https://api.github.com/repos/mozilla-services/shavar-prod-lists/commits/8ffa3c6d702002e9a357dd0042848c9418e0ef8a'
+      },
+      'protected': False
+    }
```

Comment:
Are these dictionaries temporary stubs for what will eventually come from git/GitHub?

Reply:
Yes! I copied pasted the example values from [GET API call](https://api.github.com/repos/mozilla-services/shavar-prod-lists/branches). It'll be replaced by the actual response.